### PR TITLE
Add Rancher section on /Kubernetes

### LINKED ISSUE #2438: Rancher section on /Kubernetes - build
Brief: https://docs.google.com/document/d/1MnvQ4N5xmrDKYCvbBxu3Pr7Euipk0MJ-p9luZrlkG9E/edit

Copydoc: https://docs.google.com/document/d/1G-h1eWiBSKqG9oWB15Pup8VDe1axBUwQGvtkwJ9g8T0/edit#

### PR DESCRIPTION
**Pull request by grahambancroft**
# Please don't merge until 5 December

## Done

Add Rancher section on /Kubernetes

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/kubernetes](http://0.0.0.0:8001/kubernetes)
- Make sure the copy correlates with the doc

## Issue / Card

Fixes #2438 and #2453 
Doc: https://docs.google.com/document/d/1G-h1eWiBSKqG9oWB15Pup8VDe1axBUwQGvtkwJ9g8T0/edit#

### CODE REVIEW

**Comment by grahambancroft** [comment]
@frankiegranato approves

File: templates/kubernetes/index.html [hunk 1]
```diff
@@ -35,6 +35,27 @@ <h3>Fault Tolerant</h3>
   </div>
 </section>
 
+<section class="p-strip is-bordered">
+  <div class="row p-divider">
+    <div class="col-6 p-divider__block">
+      <h2>In partnership with Google&nbsp;GKE</h2>
+      <p>Google and Canonical together enable smooth hybrid operations between Google&rsquo;s Container Engine (GKE) service with Ubuntu worker nodes, and Canonical&rsquo;s Distribution of Kubernetes. Choose Canonical&rsquo;s Kubernetes to be sure your container workloads can migrate to GKE thanks to our kernel-to-k8s&nbsp;alignment.</p>
+      <p><a href="https://cloud.google.com/container-engine/docs/node-images#ubuntu_beta" class="p-link--external">Try GKE with Ubuntu now</a></p>
```
Comment (anthonydillon) [inline templates/kubernetes/index.html]: This should be https://cloud.google.com/kubernetes-engine/docs/node-images#ubuntu. Ubuntu is now out of beta.

File: templates/kubernetes/index.html [hunk 2]
```diff
@@ -35,6 +35,27 @@ <h3>Fault Tolerant</h3>
...
+      <div class="u-align--center u-hidden--small">
+        <img src="{{ ASSET_SERVER_URL }}ba70dd8a-gce.png?h=160" alt="Google Container Engine" />
+      </div>
+    </div>
+    <div class="col-6 p-divider__block">
+      <h2>In partnership with Rancher&nbsp;Labs</h2>
+      <p>Canonical, in partnership with Rancher Labs, delivers The Cloud Native Platform: a turn-key application delivery platform, built on Ubuntu, Kubernetes, and Rancher.  Maximize developer velocity integrated CI/CD, and ease the path from development into production with enterprise-caliber container management.</p>
+      <p><a href="#" class="p-link--external" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'External Link', 'eventAction' : 'Watch webinar', 'eventLabel' : 'Watch webinar' : undefined });">Watch webinar [Need link]</a></p>
```
Comment (anthonydillon) [inline templates/kubernetes/index.html]: Just highlighting this needs a link.

File: templates/kubernetes/index.html [hunk 3]
```diff
@@ -35,6 +35,27 @@ <h3>Fault Tolerant</h3>
...
+      <div class="u-align--center u-hidden--small">
+        <img src="{{ ASSET_SERVER_URL }}8f878ba0-rancher-logo-stacked-color.svg" width="240" alt="" />
```
Comment (anthonydillon) [inline templates/kubernetes/index.html]: If there is an alt on the logo above this should either have one or the Google one should be removed for consistency.

File: templates/kubernetes/index.html [hunk 4]
```diff
@@ -35,6 +35,27 @@ <h3>Fault Tolerant</h3>
...
+      <h2>In partnership with Google&nbsp;GKE</h2>
+      <p>Google and Canonical together enable smooth hybrid operations between Google&rsquo;s Container Engine (GKE) service with Ubuntu worker nodes, and Canonical&rsquo;s Distribution of Kubernetes. Choose Canonical&rsquo;s Kubernetes to be sure your container workloads can migrate to GKE thanks to our kernel-to-k8s&nbsp;alignment.</p>
+      <p><a href="https://cloud.google.com/container-engine/docs/node-images#ubuntu_beta" class="p-link--external">Try GKE with Ubuntu now</a></p>
+      <div class="u-align--center u-hidden--small">
+        <img src="{{ ASSET_SERVER_URL }}ba70dd8a-gce.png?h=160" alt="Google Container Engine" />
+      </div>
+    </div>
+    <div class="col-6 p-divider__block">
+      <h2>In partnership with Rancher&nbsp;Labs</h2>
+      <p>Canonical, in partnership with Rancher Labs, delivers The Cloud Native Platform: a turn-key application delivery platform, built on Ubuntu, Kubernetes, and Rancher.  Maximize developer velocity integrated CI/CD, and ease the path from development into production with enterprise-caliber container management.</p>
```
Comment (anthonydillon) [inline templates/kubernetes/index.html]: Maximise not maximize.

File: templates/kubernetes/index.html (hunk 1 above)
Comment (grahambancroft) [reply templates/kubernetes/index.html]: Thank you, done and updated doc

File: templates/kubernetes/index.html (hunk 4 above)
Comment (grahambancroft) [reply templates/kubernetes/index.html]: Done. Also changed an instance in the 'Consulting packages' section

**Comment by grahambancroft** [comment]
@michaelb17 please note @anthonydillon's comment re boxy page when you review

**Comment by michaelb17** [comment]
Just looking at using another layout for the content. Thoughts? @anthonydillon @frankiegranato @grahambancroft 

screen shot 2017-11-22 at 10 35 40

**Comment by pmahnke** [comment]
@michaelb17 I think the hierarchy is confusing if you don't know which logo belongs to which company.

**Comment by michaelb17** [comment]
@pmahnke Very valid point!

**Comment by michaelb17** [comment]
I much prefer them unified under the single header, rather then the repetitive titles! +1

**Comment by spencerbygraves** [comment]
@anthonydillon We'll discuss vertical dividers in our next design meeting. With the update to the gutter widths done by @lyubomir-popov they may be unnecessary.

image

**Comment by frankiegranato** [comment]
One of the requirements of this brief is to add an anchor link to the "Rancher" portion of the page so that a "rancher specific" url can be shared in marketing campaigns. Can this be done please? (and apologies if its already been done)

**Comment by grahambancroft** [comment]
@frankiegranato Done and done, demo updated

**Comment by grahambancroft** [comment]
+1 labels removed as the design and code has changed significantly

**Comment by grahambancroft** [comment]
Please note, the image will be changing, @michaelb17 to provide new one

**Comment by grahambancroft** [comment]
[Issue to push this live on 5th December](https://app.zenhub.com/workspace/o/canonical-websites/www.ubuntu.com/issues/2463)

**Comment by frankiegranato** [comment]
aim for 11am on the 5th please

**Comment by frankiegranato** [comment]
@anthonydillon & @grahambancroft not sure which epic/milestone to add this reminder to. Any ideas?

**Pull request merged**